docs/manuals/core/entities/sequence-mgt.md

```diff
@@ -0,0 +1,111 @@
+The [Entities' creation](scenario-creation.md) section provides documentation on `Sequence^` creation. Now
+that we know how to create a new `Sequence^`, this section focuses on describing the sequence's attributes
+and utility methods.
+
+In this section, it is assumed that <a href="./code_example/my_config.py" download>`my_config.py`</a>
+module contains a Taipy configuration already implemented.
```

Comment:
```suggestion
the module contains a Taipy configuration already implemented.
```

Reply:
Or even:

"In this section, it is assumed that the \<a href="./code_example/my_config.py" download\>\`my_config.py\`\</a\>
module contains an already implemented configuration "

Reply:
Yes you are right, let me fix it, thanks.